amethyst_renderer/src/pipe/pipe.rs

```diff
@@ -233,11 +233,12 @@ where
 
         targets.insert("".into(), out.clone());
 
+        #[rustfmt::skip] // try is a reserved keyword in Rust 2018, must preserve keyword escape.
         let stages = self
             .stages
             .into_list()
             .fmap(BuildStage::new(fac, &targets, multisampling))
-            .try()?;
+            .r#try()?;
```

Comment:
Will this stay there?

Reply:
At least until rustfmt stops trying to kill it.

Reply:
try is a reserved keyword in Rust 2018, so yes that will have to stay until this function's name changes.